ntp.go

```diff
@@ -87,27 +96,38 @@ func getTime(host string, version byte) (time.Time, error) {
 
 	err = binary.Write(con, binary.BigEndian, m)
 	if err != nil {
-		return time.Now(), err
+		return nil, err
 	}
 
 	err = binary.Read(con, binary.BigEndian, m)
 	if err != nil {
-		return time.Now(), err
+		return nil, err
 	}
 
-	t := m.ReceiveTime.UTC().Local()
-	return t, nil
+	i := &Response{
+		m.Stratum,
+		m.ReceiveTime.UTC().Local(),
+	}
```

Comment:
sorry, one other minor change request.  Could you rename this variable from "i" to "r" or "resp"? That'd be a bit more idiomatic.

Thanks.